pvlib/tests/test_atmosphere.py

```diff
@@ -88,6 +88,142 @@ def test_gueymard94_pw():
     assert_allclose(pws, expected, atol=0.01)
 
 
+# Unit tests
+def test_rh_from_tdew():
+
+    # dewpoint temp calculated with who and aekr coefficients
+    dewpoint = pd.Series([
+        15.0, 20.0, 25.0, 12.0, 8.0
+    ])
+
+    # relative humidity calculated with who and aekr coefficients
+    relative_humidity_who = pd.Series([
+        72.95185312581116, 73.81500029087906, 74.6401272083123,
+        82.27063889868842, 87.39018119185337
+    ])
+    relative_humidity_aekr = pd.Series([
+        72.93876680928582, 73.8025121880607, 74.62820502423823,
+        82.26135295757305, 87.38323744820416
+    ])
+
+    temperature_ambient = pd.Series([20.0, 25.0, 30.0, 15.0, 10.0])
+
+    # Calculate relative humidity using pandas series as input
+    rh_series = atmosphere.rh_from_tdew(
+        temperature=temperature_ambient,
+        dewpoint=dewpoint
+    )
+
+    # Calulate relative humidity using pandas series as input
+    # with AEKR coefficients
+    rh_series_aekr = atmosphere.rh_from_tdew(
+        temperature=temperature_ambient,
+        dewpoint=dewpoint,
+        coeff=(6.1094, 17.625, 243.04)
+    )
+
+    # Calculate relative humidity using array as input
+    rh_array = atmosphere.rh_from_tdew(
+        temperature=temperature_ambient.to_numpy(),
+        dewpoint=dewpoint.to_numpy()
+    )
+
+    # Calculate relative humidity using float as input
+    rh_float = atmosphere.rh_from_tdew(
+        temperature=temperature_ambient.iloc[0],
+        dewpoint=dewpoint.iloc[0]
+    )
```

Comment:
It would be nice to move this down together with where the value is being tested (line 155)